libs/librepcb/projecteditor/boardeditor/fsm/bes_select.cpp

```diff
@@ -383,6 +390,9 @@ BES_Base::ProcRetVal BES_Select::proccessIdleSceneRightMouseButtonReleased(
       menu.addSeparator();
       QAction* aSelectSegment = menu.addAction(
           QIcon(":/img/actions/bookmark.png"), tr("Select Whole Segment"));
+      menu.addSeparator();
+      QAction* aMeasureSelection = menu.addAction(
+          QIcon(":/img/actions/ruler.png"), tr("Measure Selected Segment"));
```

Comment:
Somehow I'm not sure about this menu item text. With "Segment" actually LibrePCB means something different (see menu item above it: "Select Whole Segment"). I think this can be confusing for users. Maybe better call it something like "Measure Selected Traces", "Measure Traces Length", "Measure Length of Trace(s)"?

Reply:
Changed to `Measure Selected Trace Length`.